tools/ddsrouter_tool/test/compose/repeater/ddsrouter_cloud.yaml

```diff
@@ -0,0 +1,17 @@
+version: v3.0
+
+participants:
+
+  - name: Void_Participant
+    kind: void
+
```

Comment:
```suggestion
```
Not required an extra participant

Reply:
I'll leave an `echo` participant for now just to check the data is reaching the cloud router.